tests/rootfs.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/sh
+
+set -eu
+
+ERROR=1
+OK=0
+MY_NAME=$(basename -- "$0")
+ROOTFS_DIRECTORY="./rootfs"
+TEST_IMAGE_FILE_PATH="${ROOTFS_DIRECTORY}/tmp/test.img"
```

Comment:
why not store the test image in a TEST_IMAGE_FILE_PATH="${TEST_IMAGE_FILE_PATH:-$(mktmp)}"

e.g. a temporary random file by default, with override usage if running on a weird system